ui/intermittent-failures/constants.js

```diff
@@ -12,6 +12,9 @@ export const graphOneSpecs = {
   target: '',
   x_accessor: 'date',
   y_accessor: 'value',
+  // this is a temporary fix until a bug with the decimals attribute is fixed;
```

Comment:
Would you be able to add a link to that bug report here?

Reply:
Done!